collect_app/src/androidTest/java/org/odk/collect/android/formentry/FieldListUpdateTest.java

```diff
@@ -83,7 +84,7 @@
                     Manifest.permission.WRITE_EXTERNAL_STORAGE,
                     Manifest.permission.CAMERA)
             )
-            .around(new CopyFormRule(FIELD_LIST_TEST_FORM));
+            .around(new CopyFormRule(FIELD_LIST_TEST_FORM, "", Collections.singletonList("fruits.csv")));
```

Comment:
Would you consider using the existing `external-csv-search-produce.csv` data file? I think it would be appropriate to move `fieldlist-updates.xml` to the `forms` subdirectory as well. Not a big deal if you prefer to keep them separate for now.